### DISCUSSION TITLE
Welcome to GShockAPI Discussions!

### DISCUSSION THREAD

**Discussion by izivkov** [izivkov/GShockAPI#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by n3wyen** [comment]
Hello, do you have any plans to support the G Shock Move series? (ex. GBD-200) Thanks.

**Reply by izivkov** [reply]
The API should support the basic operation of this watch, except the fitness functions. Reminders are also implemented differently on this watch, via notifications, which the app does not supprt. The rest of the functions should work. You can try and see what works, and please report back.

You can also see [here](https://github.com/izivkov/CasioGShockSmartSync#supported-watches) for supposed watchers.

**Comment by jalxp** [comment]
Hey, first of all very cool project, congratulations!
I'm curious how did you started to reverse engineer the communications between the watch, did you just paired it with your watch with the official app and started some kind man in the middle to sniff what was going on, or something? How do you even start something like this?

Cheers!

**Comment by izivkov** [comment]
I used Wireshark tool to look at the ble data going between the official
app and watch. Lots of work and very tedious.

**Reply by jalxp** [reply]
I can only imagine, patience of steel! Great job 👏 
Thanks for the reply, have a good one!

**Comment by izivkov** [comment]
BTW, I'm always looking for contributors to add API for watches which I do
not have, like the some of the fitness watches with step counter. If this
is something you are interested in, let me know

**Comment by praveendaseugene** [comment]
Hi @izivkov,

I just wanted to stop by and say thank you for building and maintaining this library.

I own a GBD-200 and came across your project while exploring what's possible with G-Shock watches. Even though my specific model may not be supported yet, I was really impressed to see someone putting this much effort into open-source G-Shock tools.

Your work is appreciated. Thank you for sharing it with the community.

Best regards

**Comment by izivkov** [comment]
Hey, you are welcome.

Even though I don't have this watch and have not tried it, other people
report that the watch works with the app.

Cheers,
Ivo
